examples/FirebasePush_ESP8266/FirebasePush_ESP8266.ino

```diff
@@ -20,8 +20,8 @@
 #include <Firebase.h>
 
 // create firebase client.
-Firebase fbase("example.firebaseio.com")
-         .auth("secret_or_token");
+Firebase fbase = Firebase("example.firebaseio.com")
```

Comment:
Yes, I just restored the example the way it was before.

I think moving forward we should embrace two-step initialization, which seems to be a common pattern in Arduino-land.

```
Firebase fbase;

void setup() {
 fbase.begin("host", "auth");
}
```

WDYT?